development/golang-output-plugins.md

```diff
@@ -148,4 +148,8 @@ Note: No need to specify a plugins configuration file from command line.
 
 ```text
 fluent-bit -c fluent-bit.conf
-```
\ No newline at end of file
+```
+
+## Event Types Supported
+
+Currently, the supported event types for the output plugin are `logs` and `metrics`. These types are declared in the [flb_plugin_proxy.c](https://github.com/fluent/fluent-bit/blob/master/src/flb_plugin_proxy.c) file in the `flb_proxy_register_output` function. The `fluent-bit-go` decoder will decode both types.
```

Comment:
Can you split these over separate lines for each sentence? Makes it easier to comment on them then.

```suggestion
Currently, the supported event types for the output plugin are `logs` and `metrics`.
These types are declared in the [flb_plugin_proxy.c](https://github.com/fluent/fluent-bit/blob/master/src/flb_plugin_proxy.c) file in the `flb_proxy_register_output` function.
The `fluent-bit-go` decoder will decode both types.
```